### PR TITLE
Added Toast component and context

### PR DESCRIPTION
**Pull request by DanielBrierton**
Not entirely sure if this is how we want Toasts to work, but it seemed like a really simple option.

Any thoughts?

### CODE REVIEW

**Comment by AdrianMachado** [comment]
Can you sync this with master and include a demo video in the PR?

File: src/components/Toast/Toast.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      prefixCls,
+      isDisplayed && `${prefixCls}--displayed`,
+  );
+
+  return (
+    <h1 className={itemCls}>{text}</h1>
+  )
+};
+
+export default Toast;
```
Comment (AdrianMachado) [inline src/components/Toast/Toast.jsx]: Add a newline

File: src/components/Toast/Toast.scss [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  box-shadow: 0 6px 6px rgba(0, 0, 0, 0.3);
+  transform: translateY(-100%);
+  opacity: 0;
+  transition: opacity 0.3s ease-out, transform 0.3s ease-out;
+
+  &--displayed {
+    opacity: 1;
+    transform: translateY(0%);
+  }
+}
```
Comment (AdrianMachado) [inline src/components/Toast/Toast.scss]: newline

File: src/components/Toast/Toast.scss [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+.kai-toast {
+  background: var(--color-alpha);
+  padding: 1rem;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  color: white;
+  text-align: center;
+  z-index: var(--z-index--tab, 100000);
```
Comment (AdrianMachado) [inline src/components/Toast/Toast.scss]: nice

File: src/App.js [hunk 4]
```diff
@@ -22,118 +23,129 @@ function App() {
...
+                  header="Header text"
+                  body="body text, can support a whole lot of text"
+                />
+                <BodyTextListItem header="Header text, but no body" />
+                <ArrowListItem
+                  primary="Primary text"
+                  secondary="Secondary text"
+                />
+                <ArrowListItem primary="Just me and arrow" />
+                <Button text="Show a toast" onClick={() => { showToast(toastValues[Math.round(Math.random() * 2)], 5000) }} />
```
Comment (Phidelux) [inline src/App.js]: Why are you wrapping this in an additional arrow function?

**Comment by Phidelux** [comment]
I will merge this if @AdrianMachado has no additions. Could you include a demo video in the PR?

**Comment by DanielBrierton** [comment]
> Could you include a demo video in the PR?

There's a gif there already and it's added into the README:
https://github.com/AdrianMachado/KaiUI/blob/1aaafd99693778e8226f2e3ca291e9a2c7a52e78/component_screenshots/kaiui-toast.gif

File: src/App.js [hunk 5]
```diff
@@ -159,7 +169,6 @@ function App() {
                   secondary="Secondary text"
                 />
                 <ArrowListItem primary="Just me and arrow" />
-                <Button text="Show a toast" onClick={() => { showToast(toastValues[Math.round(Math.random() * 2)], 5000) }} />
```
Comment (Phidelux) [inline src/App.js]: I forgot to include this button in the merge. Would be great if you could fix this @DanielBrierton .

**Comment by Phidelux** [comment]
I was able to resolve my previous merge issue while merging with master. So this could finally be integrated.

**Pull request merged**